public/theme/boltcms-2020/css/style.css

```
@@ -58,19 +58,19 @@ a:hover {
 .btn-primary {
     background: #0278FC;
     border-radius: 30px;
-    color: #E7F2FF;
+    color: #ffff;
     text-align: center;
     font-weight: normal;
 }
 
-h1 {
+/* h1 {
```

```suggestion
/* h1 {
```

Should we keep this? For example, it is used on the homepage for the title :-)